ci_environment/docker/recipes/default.rb

```diff
@@ -18,10 +18,15 @@
   action :add
 end
 
-docker_pkg = 'docker-engine'
-# TODO docker_pkg += "-#{node['docker']['version']}" if node['docker']['version']
-
-package %W(linux-image-extra-#{`uname -r`.chomp} lxc #{docker_pkg})
+package %W(
+  linux-image-extra-#{`uname -r`.chomp}
```

Comment:
Oh scratch that, the vivid package already depends on extras, so even aufs should work without this! :+1: